modules/context/context_response.go

```diff
@@ -71,6 +71,7 @@ func (ctx *Context) HTML(status int, name base.TplName) {
 	if !setting.IsProd {
 		ctx.Data["TemplateName"] = name
 	}
+	ctx.Data["PageID"] = strings.Replace(string(name), "/", "-", -1)
```

Comment:
wrapperFunc: use strings.ReplaceAll method in `strings.Replace(string(name), "/", "-", -1)` (gocritic)

Reply:
Benefit of this method: initialization will be slightly faster
Drawback: If we ever rename a template, JS simply won't load which will lead to regressions.
I cannot really say if I like this drawback.

Reply:
Additionally, since IDs need to be unique, I'd prefer a prefix to avoid collisions:
```suggestion
	ctx.Data["PageID"] = "template-" + strings.Replace(string(name), "/", "-", -1)
```